include/irr/core/math/aabb2diSIMD.h

```diff
@@ -16,68 +18,13 @@ class aabb2dSIMDi
 	inline explicit aabb2dSIMDi(const core::vector4di32_SIMD& _box)
 		: internalBoxRepresentation(core::vector4di32_SIMD(-_box.x, -_box.y, _box.z, _box.w)) {};
```

Comment:
dont use the negative minimum convention for integers, due to range issues

Reply:
i know, this class will be dumped anyway